File: app/src/main/java/com/example/sample_app/CallbackActivity.java
package com.example.sample_app;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.util.Log;
import android.widget.Toast;

import com.sawolabs.androidsdk.ConstantsKt;

public class CallbackActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_callback);
        Toast.makeText(this,"Logged in successfully",Toast.LENGTH_SHORT).show();
        Intent intent = getIntent();
        String message = intent.getStringExtra(ConstantsKt.LOGIN_SUCCESS_MESSAGE);
        Log.i("payload:",message);
    }
}

File: app/src/main/java/com/sawolabs/sdkdemo/CallbackActivity.kt
package com.sawolabs.sdkdemo

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.TextView
import com.sawolabs.androidsdk.LOGIN_SUCCESS_MESSAGE
import org.json.JSONObject

class CallbackActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_callback)

        val message = intent.getStringExtra(LOGIN_SUCCESS_MESSAGE)
        findViewById<TextView>(R.id.textView).apply {
            text = message
        }
    }
}